utils/python/runflash.py

```diff
@@ -0,0 +1,3 @@
+from flash import flash
+
+flash()
```

Comment:
Can this file be removed? What is its purpose?

Reply:
It functions as an "interface layer" of sorts so that you can run the VSCode task, as for it to work with build.py it was made into a callable function rather than something you can just do `python flash.py` on. I guess a messy kinda way could be just to have the task launch a standalone Python interpreter, manually import the file and then run the method...

Reply:
Not sure I quite understand, but isn't it just a matter of changing `"command": "python utils/python/runflash.py"` to `"command": "python build.py -f"`?

Reply:
That would also build their code. If they were using CMake Tools build and then deployed, it would build a second time through build.py.

Reply:
I've managed to remove runflash.py by just having it run a command line; it's not actually that messy. See 
f15620e. Also includes a task to use build.py -f version.